pulp_docker/app/tasks/sync_stages.py

```diff
@@ -323,12 +323,14 @@ def create_blob(self, man_dc, blob_data):
             digest=digest,
         )
         blob_url = urljoin(self.remote.url, relative_url)
+        deferred_download = (self.remote.policy != Remote.IMMEDIATE)
```

Comment:
can we pass this it here https://github.com/pulp/pulp_docker/blob/master/pulp_docker/app/tasks/synchronize.py#L46
https://github.com/pulp/pulp_docker/blob/master/pulp_docker/app/tasks/sync_stages.py#L33

Reply:
I don't think we need to pass it in. I think we just need to evaluate once when initializing the FirstStage and then use that value later.